mgtool/exec.go

```diff
@@ -37,3 +39,14 @@ func run(cmd, dir string, stdout, stderr io.Writer, args ...string) error {
 
 	return c.Run()
 }
+
+func Command(path string, args ...string) *exec.Cmd {
```

Comment:
I would like to see a comment/docs for this function since it is going to be so ubiquitous.